filebeat/reader/docker_json/docker_json.go

```diff
@@ -186,3 +203,16 @@ func (p *Reader) Next() (reader.Message, error) {
 		return message, err
 	}
 }
+
+func stripNewLine(msg *reader.Message) {
```

Comment:
I'm pretty sure we have very similar code in Filebeat for stripping new lines in the new line reader.